src/components/v2v3/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx

```diff
@@ -53,22 +61,51 @@ export default function NftDrawer({
     theme,
     theme: { colors },
   } = useContext(ThemeContext)
+
+  const {
+    nftRewards: { rewardTiers: initialRewardTiers },
+  } = useContext(NftRewardsContext)
+
   const dispatch = useAppDispatch()
   const {
-    nftRewards: { rewardTiers: savedRewardTiers },
+    nftRewards,
     projectMetadata: { name: projectName, logoUri, infoUri },
   } = useAppSelector(state => state.editingV2Project)
 
+  const reduxRewardTiers = useMemo(() => nftRewards.rewardTiers, [nftRewards])
+
+  const { signer } = useWallet()
+
+  const { fundingCycleMetadata } = useContext(V2V3ProjectContext)
+
   const [marketplaceForm] = useForm<MarketplaceFormFields>()
   const [postPayModalForm] = useForm<NftPostPayModalFormFields>()
 
   const [addTierModalVisible, setAddTierModalVisible] = useState<boolean>(false)
   const [submitLoading, setSubmitLoading] = useState<boolean>(false)
 
   const [rewardTiers, setRewardTiers] = useState<NftRewardTier[]>(
-    savedRewardTiers ?? [],
+    reduxRewardTiers ?? [],
   )
 
+  useEffect(() => {
```

Comment:
This is not ideal, I know. Was battling trying to load the `reduxRewardTiers ` into the `rewardTiers` state. Line 88 wasn't doing the job (`rewardTiers` state remains empty).